integrationTests/p2p/antiflood/nilQuotaStatusHandler.go

```diff
@@ -0,0 +1,18 @@
+package antiflood
+
+type nilQuotaStatusHandler struct {
+}
+
+// ResetStatistics is not implemented
+func (nqsh *nilQuotaStatusHandler) ResetStatistics() {
+}
+
+// AddQuota is not implemented
+func (nqsh *nilQuotaStatusHandler) AddQuota(identifier string, numReceivedMessages uint32, sizeReceivedMessages uint64,
```

Comment:
new line for every input param

Reply:
Kept them on the same line, replaced parameters name with _ placeholder